src/components/SingInForm.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react";
+import { useDispatch } from "react-redux";
+import { toast } from "react-toastify";
+import { Button, Form, FormInput } from "semantic-ui-react";
+import { singInRequest } from "../api";
+import { setAutorizationStatusAction } from "../redux/actions/autorizationAction";
+import { Props } from "./SingUpForm";
+
+export default function SingInForm(props: Props): JSX.Element {
+  const dispatch = useDispatch();
+  const [userLogin, setUserLogin] = useState("");
+  const [userPassword, setUserPassword] = useState("");
+
+  async function checkUserPassword() {
+    const userData = await singInRequest(userLogin, userPassword);
+
+    if (userData.data.length) {
+      dispatch(setAutorizationStatusAction());
```

Comment:
setAutorizationStatusAction is useless.
You can use userData as indicator for authorizationStatus

Reply:
fixed